docs/guide/api-environment-frameworks.md

```diff
@@ -186,10 +190,10 @@ function vitePluginVirtualIndexHtml(): Plugin {
         let html: string
         if (server) {
           this.addWatchFile('index.html')
-          html = await fs.promises.readFile('index.html', 'utf-8')
+          html = await readFile('index.html', 'utf-8')
```

Comment:
I don't know if I like named importing `fs` 😅  Perhaps we can use `fs.readFileSync` instead to shorten it? And avoid the import?

Reply:
Ah, yes, good idea, done 👍🏼